helm/cluster-vsphere/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 1
+description: "A helm chart for creating Cluster API clusters with the VSphere provider (CAPV)."
+keywords:
+- capv
+- vsphere
+kubeVersion: ">=1.19.0 < 1.23.0"
+maintainers:
```

Comment:
Should also have a team annotation (I think this is used for alert routing). See https://github.com/giantswarm/cluster-openstack/blob/main/helm/cluster-openstack/Chart.yaml#L7-L8 for example.